tests/functional/scenario1/DatadirScenarioTest.php

```diff
@@ -500,6 +519,21 @@ public function testUser2HaveCorrectAccessAfterProvisioning(): void
         $this->assertCount(2, $user2TableRows);
     }
 
+    public function testUserResetPassword(): void
+    {
+        $configArray = self::getUserResetPasswordConfig();
+        $config = $this->getConfigFromConfigArray($configArray);
+
+        $connection = $this->getConnectionForConfig($config);
+
+        $this->runAppWithConfig($configArray);
+
+        $userName = $this->namingConventions->getUsernameFromEmail($config->getUser());
```

Comment:
Tohle testuje něco jiného - to testuje ten db adapter (který nota bene časem zmizne do db-adapter-snowflake)

Správný test by vytvořil uživatele (runAppWithConfig), zkusil se připojit. Zjistil, že to funguje. Pak by poslal request na reset hesla (tj. runAppWithConfig jen s resetPassword navíc) a zkontroloval by, že původní údaje vyhodí výjimku. 

Jako bonus points by to mohlo přidat do runAppWithConfig vrácení toho processu a assertnout na tom outputu že tam je ten reset hesla.

Reply:
Jo, já zapomněl, že to vrací ten reset url a ne že by to přímo resetlo to původní heslo. Tím pádem není možné testovat, že se dá/nedá připojit. Ale bude jen stačit přidat to vrácení processu z runAppWithConfig

> Executing the ALTER USER … RESET PASSWORD statement does not invalidate the current password. The user can continue to use the old password until the new password is set.

Reply:
Možná mě napadá, že by bylo fajn ještě doplnit do toho logu do message "old password will keep working until you reset the password using provided link" nebo tak něco.

Reply:
A poslední věc ještě - že možná by se tenhle test dal přilepit k některému z ostatních, které stejně dělají uživatele - tak, že by sis ušetřil zbytečné smazání+vytvoření, které je tam navíc.

Reply:
doplnil jsem tu message a přilepil jsem to k jednomu testu který tam už je - musel jsem vytáhnout process z `runAppWithConfig` jak si psal